ts/src/base/Exchange.ts

```diff
@@ -5783,7 +5783,7 @@ export default class Exchange {
         return this.filterByArray (results, 'symbol', symbols);
     }
 
-    parseTickers (tickers, symbols: string[] = undefined, params = {}): Dictionary<Ticker> {
+    parseTickers (tickers: DictOrDictList, symbols: string[] = undefined, params = {}): Dictionary<Ticker> {
```

Comment:
again, I think the approach needs to be different here; instead of introducing these exotic types, why not accept only a List and update the implementations providing a dict?

Reply:
> again, I think the approach needs to be different here; instead of introducing these exotic types, why not accept only a List and update the implementations providing a dict?

so that we would have 2 different `parseTickers`methods?

Reply:
or we can just convert from dict to list before calling `parseTickers` inside the exchange-specific implementation (I think most exchanges provide a list here)

Reply:
> or we can just convert from dict to list before calling `parseTickers` inside the exchange-specific implementation (I think most exchanges provide a list here)

That's a lot more code, why not just create a type like this

Reply:
or maybe we could have a type like `DictOrList`

Reply:
@samgermain because that will not age well and it does not make the code much more strict, it's easier to maintain a method that can only receive a list or a dict and not both, simpler is better in this case